torch/_dynamo/variables/builder.py

```diff
@@ -671,6 +673,17 @@ def index_source(key):
             return self.tx.output.side_effects.track_object_existing(
                 self.source, value, result
             )
+        elif isinstance(value, types.GetSetDescriptorType):
+            return GetSetDescriptorVariable(
+                value, guards=self.make_guards(GuardBuilder.ID_MATCH)
+            )
+        elif isinstance(value, types.MethodWrapperType) and value.__name__ == "__get__":
+            return GetAttrFunctionVariable(
```

Comment:
it's not super useful - we can't trace into it and we'll basically only get attrs from it at least in the torch function work. If you see other uses for it I can add it, I just didn't think it was needed.

Reply:
> we can't trace into it

Isn't this precisely the situation when we need to have a variable tracker for it. If you model this as MethodWrapperType you have a logical place to hang off more implementations if you need to hit other wrapper types as well. Is it that much worse? Surely it can't be...